app/helpers/number_helper.rb

```diff
@@ -69,8 +69,6 @@ def mhz_to_human_size(size, *args)
     nil
   end
 
-  private
-
```

Comment:
So, I'm weird and maybe I don't agree with rubocop here, but I *like* have the private notifier here because it tells me everything under here is private.  Yeah, you still need to make class methods private, separately but it gives that extra visual separation.  ¯\\\_(ツ)_/¯  What do you think?